notebooks/en/enterprise_cookbook_overview.md

```diff
@@ -0,0 +1,50 @@
+# Enterprise Hub Cookbook
+
+The Enterprise Hub Cookbook is designed for power users and enterprises who want to go beyond the standard free features of the Hugging Face Hub and integrate machine learning deeper into their production workflows. 
+
+The cookbook guides you through a selection of recipes (Jupyter Notebooks) with copy-pastable code to help you get started with advanced features of the Hub. For a quick overview of the Enterprise Hub, you can also watch [this 15 minute video](https://www.youtube.com/watch?v=CPQGBn-yXJQ). 
+
+
+## 1. Interactive Development in HF Spaces [coming soon]
```

Comment:
I think it might be better not to do an ordered list since this might give users the impression that they should follow these recipes in a certain order.

```suggestion
## Interactive Development in HF Spaces [coming soon]
```